pkg/analysis/statussubresource/analyzer.go

```diff
@@ -158,3 +152,7 @@ func hasStatusField(sTyp *ast.StructType, jsonTags extractjsontags.StructFieldTa
 
 	return false
 }
+
+func kubebuilderFormatWithValue(value string) string {
```

Comment:
This should probably mention that it's the root marker, or have an option to pass in a marker?

Reply:
These formatted functions are also unified as Format.